examples/chat/api/index.js

```diff
@@ -1,13 +1,13 @@
 const data = require('./mock-data')
 const LATENCY = 16
 
-export function getAllMessages (cb) {
+export const getAllMessages = (cb) => {
   setTimeout(() => {
     cb(data)
   }, LATENCY)
 }
 
-export function createMessage ({ text, thread }, cb) {
+export const createMessage = ({ text, thread }, cb) => {
```

Comment:
Hmm, I don't think we should always use arrow function since function declaration and arrow function is different.